src/general/PauseMenu.cs

```diff
@@ -49,11 +49,55 @@ public class PauseMenu : ControlWithInput
     [Signal]
     public delegate void MakeSave(string name);
 
+    private enum MenuEnum
```

Comment:
```suggestion
    private enum ActiveMenuType
```
I don't like using Enum as a suffix...